tianshou/exploration/random.py

```diff
@@ -52,32 +50,39 @@ class OUNoise(BaseNoise):
     Ornstein-Uhlenbeck process.
     """
 
-    def __init__(self,
-                 mu: float = 0.0,
-                 sigma: float = 0.3,
-                 theta: float = 0.15,
-                 dt: float = 1e-2,
-                 x0: Optional[Union[float, np.ndarray]] = None
-                 ) -> None:
-        super(BaseNoise, self).__init__()
+    def __init__(
+        self,
+        mu: float = 0.0,
+        sigma: float = 0.3,
+        theta: float = 0.15,
+        dt: float = 1e-2,
+        x0: Optional[Union[float, np.ndarray]] = None,
+    ) -> None:
+        super().__init__()
         self._mu = mu
         self._alpha = theta * dt
         self._beta = sigma * np.sqrt(dt)
         self._x0 = x0
         self.reset()
 
-    def __call__(self, size: tuple, mu: Optional[float] = None) -> np.ndarray:
+    def reset(self) -> None:
+        """Reset to the initial state."""
+        self._x = self._x0
+
+    def __call__(
+        self, size: Sequence[int], mu: Optional[float] = None
+    ) -> np.ndarray:
         """Generate new noise. Return a ``numpy.ndarray`` which size is equal
         to ``size``.
         """
-        if self._x is None or self._x.shape != size:
-            self._x = 0
+        if (
+            self._x is None
```

Comment:
Strange writting.

Reply:
https://github.com/psf/black/blob/master/docs/the_black_code_style.md#parentheses

Reply:
so, what's the point ?

Reply:
I use black to format only a little parts of the code, here is the formatter’s result...